easybuild/easyconfigs/n/nodejs/nodejs-12.16.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'nodejs'
+version = '12.16.1'  # 12.16.1 was LTS on 2020-05-22
+local_libversion = '72'
+
+homepage = 'http://%(name)s.org'
+description = """Node.js is a platform built on Chrome's JavaScript runtime
+ for easily building fast, scalable network applications. Node.js uses an
+ event-driven, non-blocking I/O model that makes it lightweight and efficient,
+ perfect for data-intensive real-time applications that run across distributed devices."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['http://%(name)s.org/dist/v%(version)s/']
+sources = ['node-v%(version)s.tar.gz']
+checksums = ['4fe8c3454f9bee5bbe72d44aa25cd931859b3037b7a9473081b3b2bd1b465b95']
+
+builddependencies = [('binutils', '2.32')]
+
+# Python is required (only) as build dependency
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
```

Comment:
Since `Python` is also installed with `GCCcore`, we might as well list `Python` (2.7.16) as a proper build dependency?

That way we're not at the mercy of the OS, which may provide a very old Python (2.6), or a very recent one (3.8), which could break the build.